crates/next-core/src/next_config.rs

```diff
@@ -908,63 +908,6 @@ pub struct ExperimentalConfig {
     devtool_segment_explorer: Option<bool>,
 }
 
-#[derive(
-    Clone, Debug, PartialEq, Serialize, Deserialize, TraceRawVcs, NonLocalValue, OperationValue,
-)]
-#[serde(rename_all = "camelCase")]
-pub struct CacheLifeProfile {
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub stale: Option<u32>,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub revalidate: Option<u32>,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub expire: Option<u32>,
-}
```

Comment:
We don't actually care what the contents of `CacheLifeProfile` are, only that we can round-trip it. So `serde_json::Value` seems like a much simpler/better choice.